rules/windows/execution_via_hidden_shell_conhost.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/08/17"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects when the Console Window Host (conhost.exe) process is spawned by a suspicious parent process, which could be indicative of code injection.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "conhost.exe Spawned By Suspicious Parent Process"
```

```suggestion
name = "Conhost Spawned By Suspicious Parent Process"
```
I wouldn't put the `.exe` in the rule name and I'd capitalize for consistency.